tests/test_news_flash.py

```diff
@@ -9,8 +9,16 @@
 from anyway.parsers.news_flash_db_adapter import init_db
 from anyway.models import NewsFlash
 from anyway.parsers import timezones
+from anyway.parsers.infographics_data_cache_updater import is_cache_eligible, is_in_cache
 
 
+def are_in_cache(news_flash_list):
+    for nf in news_flash_list:
+        if is_cache_eligible(nf) and not is_in_cache(nf):
+            print(f"NewsFlash {nf.get_id()} not in cache")
```

Comment:
Instead of printing, I believe it's better to assert here:
```python
def assert_all_in_cache(news_flash_list):
    for nf in news_flash_list:
        if is_cache_eligible(nf):
            assert is_in_cache(nf), "NewsFlash {nf.get_id()} not in cache"
```
and remove the assertion on call site.

Reply:
My thought was to provide better context of the flow that reached the error, but I understand that the information from the assert provides it anyway. OK.